CRM/Utils/Hook.php

```diff
@@ -1474,6 +1474,20 @@ public static function triggerInfo(&$info, $tableName = NULL) {
       'civicrm_triggerInfo'
     );
   }
+  /**
+   * This hook allows changes to the spec of which tables to log.
+   *
+   * @param array $logTableSpec
+   *
+   * @return mixed
+   */
+  public static function logTableSpec(&$logTableSpec) {
```

Comment:
IMHO, `hook_logTables` or `hook_alterLogTables` would be more consistent with the other hook names.